addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -18,9 +18,18 @@ def send_mail_test(self):
         ctx.pop('default_state', None)
         self = self.with_context(ctx)
 
+        notification_messages = []
         mails_sudo = self.env['mail.mail'].sudo()
+        test_emails = []
+        for candidate in self.email_to.split(','):
```

Comment:
What if my email_to is '"Charles, the beast from Windsor, Prince of Walles" <charles@example.uk' ? This one is a valid email and should not be considered as two (or three).